Package.swift

```diff
@@ -411,10 +411,6 @@ let package = Package(
             dependencies: [
                 "Common"
             ],
-            resources: [
-                .copy("hashPrefixes.json"),
-                .copy("filterSet.json")
-            ],
```

Comment:
I checked this with @not-a-rootkit, this change is trying to copy files which don't exist on-disk which causes problems with Xcode 16 - we can safely remove them.

Reply:
Thanks, these warnings were becoming annoying :)